src/pages/AppList.svelte

```diff
@@ -1,4 +1,7 @@
 <script>
+	import { fade } from 'svelte/transition';
+	let visible = true;
```

Comment:
We should use a more descriptive variable name here.

```suggestion
	let showDeprecated = true;
```